tests/System/Routes/ConfirmSubscriptionRouteTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace WMDE\Fundraising\Frontend\Tests\System\Routes;
+
+use WMDE\Fundraising\Entities\Subscription;
+use WMDE\Fundraising\Frontend\FunFunFactory;
+use WMDE\Fundraising\Frontend\Tests\Fixtures\SubscriptionRepositorySpy;
+use WMDE\Fundraising\Frontend\Tests\System\WebRouteTestCase;
+use WMDE\Fundraising\Frontend\Messenger;
+use Swift_NullTransport;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Gabriel Birke < gabriel.birke@wikimedia.de >
+ */
+class ConfirmSubscriptionRouteTest extends WebRouteTestCase {
+
+	// @codingStandardsIgnoreStart
+	protected function onTestEnvironmentCreated( FunFunFactory $factory, array $config ) {
+		// @codingStandardsIgnoreEnd
+		$factory->setMessenger( new Messenger(
+			Swift_NullTransport::newInstance(),
+			$factory->getOperatorAddress()
+		) );
+	}
+
+	public function testGivenAnUnconfirmedSubscriptionRequest_successPageIsDisplayed() {
```

Comment:
If you have this, then I'd also expect to have a `testGivenAConfirmedSubscriptionRequest_stuffHappens`. Or does that not make sense?